src/operator/softmax_output.cc

```diff
@@ -30,59 +30,77 @@ Operator *SoftmaxOutputProp::CreateOperatorEx(Context ctx, std::vector<TShape> *
 DMLC_REGISTER_PARAMETER(SoftmaxOutputParam);
 
 MXNET_REGISTER_OP_PROPERTY(SoftmaxOutput, SoftmaxOutputProp)
-.describe(R"code(Softmax with logit loss.
+.describe(R"code(Computes the gradient of cross entropy loss with respect to softmax output.
 
-In the forward pass, the softmax output is returned. Assume the input data has
-shape *(n,k)*, then the output will have the same shape as the input, which is computed by
+- This operator is finished in two steps.
```

Comment:
Same issue - "is finished" is probably not the right language here. See above comment